src/main/java/io/cdap/plugin/orientdb/OrientDBConfig.java

```diff
@@ -78,4 +101,134 @@ public String getVertexType() {
   public String getEdgeType() {
     return edgeType;
   }
+
+  public void validate(FailureCollector failureCollector, Schema inputSchema) {
+    try {
+      IdUtils.validateId(referenceName);
```

Comment:
FYI: the newest version of hydrator-commons has a new IdUtils method that takes a failureCollector